Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "zcash_script"
-version = "0.1.7"
+version = "0.1.9"
```

Comment:
Did this need to be updated or should it be left as `0.1.7`?

Reply:
We should update it, weird that it worked before :thinking: